src/analytics/loadingPage/useLogDownloadAnalytics.ts

```diff
@@ -13,11 +14,11 @@ interface P extends Properties {}
 interface Analytics extends A<Action> {}
 
 export const useLogDownloadAnalytics = (): Analytics => {
-  const sendEvent: Analytics["sendEvent"] = (action) => {
+  const sendEvent: Analytics["sendEvent"] = useCallback((action) => {
```

Comment:
These hooks are all so similar, do you think it would be worth creating a reusable base analytics hook that takes in the `object` string? Understandable if that seems like overkill though!

Reply:
What a great suggestion! Done. What do you think?